lib/libc/riscv/gen/_setjmp.S

```diff
@@ -141,13 +226,18 @@ ENTRY(_longjmp)
 	/* Load the return value */
 	mv	a0, a1
 	ret
+#endif
 
 botch:
 #ifdef _STANDALONE
 	j	botch
+#elif defined(__CHERI_PURE_CAPABILITY__)
+	clgc	ct0, _C_LABEL(longjmperror)
```

Comment:
Don't we have CALL macros? The implementation here can probably be shared.

Reply:
Oh, perhaps.  I can see if I can use some existing macros.  I find the use of too many macros (ala MIPS assembly) to be obfuscating, but if there are simple CALL wrappers that work I can try those.